Exercise_A/assets/css/style.css

```diff
@@ -1,3 +1,491 @@
+         background-color: #fff;
+     }
+ 
+     .navbar-default.affix .navbar-header .navbar-brand {
+         font-size: 14px;
+         color: #f05f40;
+     }
+ 
+     .navbar-default.affix .navbar-header .navbar-brand:hover,
+     .navbar-default.affix .navbar-header .navbar-brand:focus {
+         color: #eb3812;
+     }
+ 
+     .navbar-default.affix .nav > li>a,
+     .navbar-default.affix .nav>li>a:focus {
+         color: #222;
+     }
+ 
+     .navbar-default.affix .nav > li>a:hover,
+     .navbar-default.affix .nav>li>a:focus:hover {
+         color: #f05f40;
+     }
+ }
+ 
+ header {
+     position: relative;
+     width: 100%;
+     min-height: auto;
+     text-align: center;
+     color: #fff;
+     background-image: url(/Exercise_A/assets/images/header.jpg);
+     background-position: center;
+     -webkit-background-size: cover;
+     -moz-background-size: cover;
+     background-size: cover;
+     -o-background-size: cover;
+ }
+ 
+ header .header-content {
+     position: relative;
+     width: 100%;
+     padding: 100px 15px;
+     text-align: center;
+ }
+ 
+ header .header-content .header-content-inner h1 {
+     margin-top: 0;
+     margin-bottom: 0;
+     text-transform: uppercase;
+     font-weight: 700;
+ }
+ 
+ header .header-content .header-content-inner hr {
+     margin: 30px auto;
+ }
+ 
+ header .header-content .header-content-inner p {
+     margin-bottom: 50px;
+     font-size: 16px;
+     font-weight: 300;
+     color: rgba(255,255,255,.7);
+ }
+ 
+ @media(min-width:768px) {
+     header {
+         min-height: 100%;
+     }
+ 
+     header .header-content {
+         position: absolute;
+         top: 50%;
+         padding: 0 50px;
+         -webkit-transform: translateY(-50%);
+         -ms-transform: translateY(-50%);
+         transform: translateY(-50%);
+     }
+ 
+     header .header-content .header-content-inner {
+         margin-right: auto;
+         margin-left: auto;
+         max-width: 1000px;
+     }
+ 
+     header .header-content .header-content-inner p {
+         margin-right: auto;
+         margin-left: auto;
+         max-width: 80%;
+         font-size: 18px;
+     }
+ }
+ 
+ @media(min-width:1200px) {
+   .no-gutter .col-lg-4 {
+     width: 25%;
+   }
+ }
+ 
+ .section-heading {
+     margin-top: 0;
+ }
+ 
+ .service-box {
+     margin: 50px auto 0;
+     max-width: 400px;
+ }
+ 
+ @media(min-width:992px) {
+     .service-box {
+         margin: 20px auto 0;
+     }
+ }
+ 
+ .service-box p {
+     margin-bottom: 0;
+ }
+ 
+ .portfolio-box {
+     display: block;
+     position: relative;
+     margin: 0 auto;
+     max-width: 650px;
+ }
+ 
+ .portfolio-box-caption {
+     display: block;
+     position: absolute;
+     bottom: 0;
+     width: 100%;
+     height: 100%;
+     text-align: center;
+     color: #fff;
+     opacity: 0;
+     background: rgba(240,95,64,.9);
+     -webkit-transition: all .35s;
+     -moz-transition: all .35s;
+     transition: all .35s;
+ }

```

Comment:
If you use flex and responsive sizing you will not need to handle this for every @ media query